lib/airports.rb

```diff
@@ -12,6 +12,19 @@ def self.find_by_iata_code(iata_code)
     Airport.
       new(airport_data.each_with_object({}) { |(k, v), h| h[k.to_sym] = v })
   end
+  
+  def self.find_by_city(city)
+    return unless city.length < 1
```

Comment:
I'd ✂️ this - not really any need for it I think.

Reply:
make sense